.github/workflows/check-rebaseable.yml

```diff
@@ -16,21 +16,19 @@ jobs:
   check-rebaseable:
     runs-on: ubuntu-latest
     env:
-      GH_TOKEN: ${{ secrets.DETERMINED_TOKEN }}
-      GITHUB_TOKEN: ${{ secrets.DETERMINED_TOKEN }}
       EE_REPO: determined-ai/determined-ee
+      OSS_REPO: determined-ai/determined
     steps:
-      - name: get PR
+      - name: get EE
         uses: actions/checkout@v3
         with:
-          fetch-depth: 1  # we only need this branch
+          fetch-depth: 1  # we only need the main branch
           token: ${{ secrets.DETERMINED_TOKEN }}
+          repository: ${{ EE_REPO }}
       - name: add EE repository remote and get default branch
         run: |
-          DEFAULT_BRANCH=$( gh api repos/${{ env.EE_REPO }} -q '.default_branch' )
-          git remote add EE github.com:${{ env.EE_REPO }}
-          git fetch EE
-          git checkout EE/$DEFAULT_BRANCH
+          git remote add OSS github.com:${{ env.OSS_REPO }}
```

Comment:
PRs may come from a fork -- we should use the event to get the source repo

Reply:
Doh!  You're right.  That's the whole reason for this pull_request_target thing. :D

Reply:
Wait, no.  The github_ref will be refs/pull/1234 on a PR, so if it's a pull from a fork, that's fine because there is a new ref in the OSS repo for that pull.  This is fine (.gif).

Reply:
```
sauer@host:~/determined/determined-ee$ git fetch oss refs/pull/5364/head:somebranch
remote: Enumerating objects: 19, done.
remote: Counting objects: 100% (19/19), done.
remote: Compressing objects: 100% (7/7), done.
remote: Total 15 (delta 12), reused 11 (delta 8), pack-reused 0
Unpacking objects: 100% (15/15), 3.39 KiB | 693.00 KiB/s, done.
From github.com:determined-ai/determined
 * [new ref]             refs/pull/5364/head -> somebranch
sauer@host:~/determined/determined-ee$ git rebase somebranch 
... (pretend this works :D)...
```